R/assess_security.R

```diff
@@ -0,0 +1,62 @@
+#' Assess a package for known security vulnerabilities in the OSS Index
+#'
+#' @param x a \code{pkg_ref} package reference object
+#' @param ... additional arguments passed on to S3 methods, rarely used
+#' @return a \code{pkg_metric} containing Assess for any known security vulnerabilities in the OSS Index via oysteR
+#' @seealso \code{\link{metric_score.pkg_metric_security}}
+#'
+#' @importFrom utils install.packages menu
+#' @export
+assess_security <- function(x, ...) {
+  # TODO: discuss preferred approach for handling packages within Suggests
+  if (!requireNamespace("oysteR", quietly = TRUE)) {
```

Comment:
Does the version of oysteR they have installed matter?

Reply:
> Does the version of oysteR they have installed matter?

Good catch. Addressed in 1a7e7da

> Also, more of a general question is if we need to create some tests for each new metric.

A good discussion to have generally. I did plan on adding them for this metric once the `Suggests` implementation details were sorted out.